src/meteors/shap/shap.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+import shap
+import numpy as np
+import pandas as pd
+import torch
+
+from .explanation import SHAPExplanation, AVAILABLE_SHAP_EXPLAINERS
+
+from meteors.models import ExplainableModel
+
+from typing import Literal
+
+
+class HyperSHAP:
+    def __init__(
+        self,
+        callable: ExplainableModel,
+        masker,
+        explainer_type: Literal["exact", "tree", "kernel", "linear"] | None = None,
```

Comment:
Why None? What for?

Reply:
updated in ac31a16ee92f0ce12b39d88da40cd1329e6c9220